build.wake

```diff
@@ -60,20 +61,20 @@ global def runFreedomMetalInstall options =
 
   def installedFreedomMetal =
     def inputs =
-      sources "freedom-metal" `.*`
-      | filter (!matches `freedom-metal/doc/.*` _.getPathName)
+      sources freedomMetalRoot `.*`
+      | filter (!matches "{freedomMetalRoot}/doc/.*".quote _.getPathName)
```

Comment:
I think the `quote` needs to be called on a subset of the string that exclude the regular expression special characters like `.` and `*` because otherwise they would get escaped ("quoted") as literal characters. You can quote this variable part and then call [`regExpCat`](https://github.com/sifive/wake/blob/v0.19.1/share/wake/lib/core/regexp.wake#L28) in order to concatenate two RegExp objects, e.g. ``regExpCat freedomMetalRoot.quote `/doc/.*` ``.

I'm not 100% certain about this, so you might want to test out this behavior.

Reply:
Now using `stringToRegExp`

Reply:
Actually that returns a `Result` for a reason I can't understand